packages/visualizations/addon/templates/components/visualization-config/table.hbs

```diff
@@ -1,4 +1,16 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#if (check-config 'environment' 'development')}}
+  <div class='table-config__totals-toggle-section'>
```

Comment:
Can we use more generic class names? This is not a `totals-toggle`.

Reply:
Maybe change to `toggle-section`?